lib/matplotlib/backend_tools.py

```diff
@@ -335,6 +335,16 @@ def send_message(self, event):
             except (ValueError, OverflowError):
                 pass
             else:
+                artists = [a for a in event.inaxes.mouseover_set
+                           if a.contains(event) and a.get_visible()]
+
+                if artists:
+                    a = max(enumerate(artists), key=lambda x: x[1].zorder)[1]
```

Comment:
My guess is that it is but to be honest, I didn't check, I just copy/paste from `backend_bases.py`